docs/src/features/pdf-export.md

```diff
@@ -15,7 +15,15 @@ After you've installed _weasyprint_, run _presenterm_ with the `--export-pdf` pa
 presenterm --export-pdf examples/demo.md
 ```
 
-The output PDF will be placed in `examples/demo.pdf`. 
+The output PDF will be placed in `examples/demo.pdf`.
```

Comment:
Can you remove these doc changes? This gets automatically pushed on merge and will otherwise be misleading since this isn't released yet